css/index.css

```diff
@@ -1,3 +1,236 @@
+* {
+	box-sizing: border-box;
+}
 
+html, body {
+	height: 100%;
+	font-family: 'Titillium Web', sans-serif;
+}
 
+h1, h2, h3, h4, h5 {
+	font-family: 'Bangers', cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
+}
+
+
+body {
+color: #232323;
+}
+
+.container {
+	max-width: 910px;
+	padding: 0 15px;
+	width: 100%;
+	margin:	 0 auto;
+}
+
+@media (max-width: 800px) {
+	.container {
+	}
+}
+
+@media (max-width: 500px) {
+	.container {
+		padding: 0px;
+	}
+}
+
+.nav {
+padding: 30px 0;
+background-color: white;
+display: flex;
+align-items: baseline;
+color:	gray;
+}
+
+@media (max-width: 800px) {
+	.nav {
+		display: flex;
+		flex-direction: row-reverse;
+		flex-wrap: wrap-reverse;
+		padding: 0px;
+	}
+}
+
+@media (max-width: 500px) {
+	.nav {
+		padding: 5px;
+	}
+}
+
+.nav-links {
+	padding: 0px;
+	width: 100%;
+	display: flex;
+	justify-content: space-evenly;
+}
+
+@media (max-width: 800px) {
+	.nav-links {
+		flex-wrap: wrap;
+		width: 100%;
+		justify-content: space-evenly;
+	}
+}
+
+@media (max-width: 500px) {
+	.nav-links {
+		width: 100%;
+		flex-direction: column;
+		align-items: center;
+	}
+}
+
+@media (max-width: 500px) {
+	a {
+		display: flex;
+		width: 100%;
+		border-bottom: .5px solid black;
+		border-top: .5px solid black;
+		height: 50px;
+		justify-content: center;
+		align-items: center;
+	}
+}
+@media (max-width: 800px) {
+	.main-logo {
+		width: 100%;
+		justify-content: center;
+		display: flex;
+		margin: 20px 0px 75px 0px;
+	}
+}
+
+.logo {
+	padding: 0px;
+}
+
+@media (max-width: 800px) {
+	.logo {
+		display: flex;
+		height: 30px;
+	}
+}
+
+.nav a {
+text-decoration: none;
+color: gray;
+transform: translateY(-10px);
+}
+
+.header {
+	display: flex;
+	width: 100%;
+	align-items: center;
+	justify-content: space-evenly;
+}
+
+.heading-wrapper {
+text-align: center;
+display: inline-block;
+padding: 50px;
+width: 100%;
+}
+
+@media (max-width: 500px) {
+	.heading-wrapper {
+		padding: 10px;
+	}
+}
+
+.heading {
+font-size: 72px;
+margin-bottom: 20px;
+}
+
+.img-wrapper {
+	align-content: center;
+}
+
+@media (max-width: 800px) {
+	.img-wrapper {
+		display: none;
+	}
+}
+
+@media (max-width: 500px) {
+	.img-wrapper {
+		display: none;
+	}
+}
+
+.group:before, .group:after {
+	content: "";
+	display: table;
+}
+
+.group:after {
+clear: both;
+}
+
+.btn {
+display: inline-block;
+text-decoration: none;
+color: inherit;
+padding: 8px 60px;
+border: 1px solid black;
+}
+
+.top-section,
+.bottom-section,
+.contact-section {
+padding: 30px 0;
+text-align: justify;
+width: 100%
+}
+
+@media (max-width: 500px) {
+	.top-section {
+		width: 100%;
+	}
+}
+
+.top-section article {
+display: inline-block;
+width: 48%;
+padding-right: 20px;
+}
+
+@media (max-width: 500px) {
+	.top-section article {
+		display: flex;
+		flex-direction: column;
+		width: 100%;
+	}
+}
+
+.bottom-section article {
+display: inline-block;
+width: 32.81%;
+padding-right: 20px;
+}
+
```

Comment:
you could utilize flex-box throughout the site instead of inline, then in the media queries you would just need to change the flex-direction instead of having the container as inline-block in desktop and switching it to flex in the smaller breakpoints, this would reduce code and make manipulating them easier in all the layouts